lib/dao.js

```diff
@@ -148,7 +148,8 @@ function noCallback(err, result) {
  *   - instance (null or Model)
  */
 DataAccessObject.create = function (data, options, cb) {
-  if (stillConnecting(this.getDataSource(), this, arguments)) return;
+  var p;
+  if (p = stillConnecting(this.getDataSource(), this, arguments)) return p;
```

Comment:
Hehe...I just copied the code from your previous comment :)
I had to pull the `var p` out of the `if` statement....but yeah, I definitely prefer this style too.